tests/e2e/scenarios/scalability/run-test.sh

```diff
@@ -86,12 +65,25 @@ echo "ADMIN_ACCESS=${ADMIN_ACCESS}"
 # cilium does not yet pass conformance tests (shared hostport test)
 #create_args="--networking cilium"
 create_args=()
-create_args=("--network-cidr=10.0.0.0/16,10.1.0.0/16,10.2.0.0/16,10.3.0.0/16,10.4.0.0/16,10.5.0.0/16,10.6.0.0/16,10.7.0.0/16,10.8.0.0/16,10.9.0.0/16,10.10.0.0/16,10.11.0.0/16,10.12.0.0/16")
+if [[ "${CLOUD_PROVIDER}" == "aws" ]]; then
+  create_args+=("--network-cidr=10.0.0.0/16,10.1.0.0/16,10.2.0.0/16,10.3.0.0/16,10.4.0.0/16,10.5.0.0/16,10.6.0.0/16,10.7.0.0/16,10.8.0.0/16,10.9.0.0/16,10.10.0.0/16,10.11.0.0/16,10.12.0.0/16")
+  create_args+=("--node-size=t3a.medium,t3.medium,t2.medium,t3a.large,c5a.large,t3.large,c5.large,m5a.large,m6a.large,m5.large,c4.large,c7a.large,r5a.large,r6a.large,m7a.large")
+  create_args+=("--node-volume-size=20")
+  create_args+=("--zones=us-east-2a,us-east-2b,us-east-2c")
+  create_args+=("--image=${INSTANCE_IMAGE:-ssm:/aws/service/canonical/ubuntu/server/20.04/stable/current/amd64/hvm/ebs-gp2/ami-id}")
+fi
+if [[ "${CLOUD_PROVIDER}" == "gce" ]]; then
+  create_args+=("--zones=us-east1-b,us-east1-c,us-east1-d")
+  create_args+=("--node-size=e2-standard-2")
+  create_args+=("--node-volume-size=30")
+  create_args+=("--master-volume-size=1000")
```

Comment:
Interesting to see the volume size to be `1000`, out of curiosity, why would it need `1000` ?

Reply:
Bigger disks have more iops on GCE